src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/volume/vanilla/PotionFluidVolume.java

```diff
@@ -0,0 +1,19 @@
+package io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.vanilla;
+
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.math.Fraction;
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.ImmutableFluidVolume;
+
+import net.minecraft.fluid.Fluids;
+import net.minecraft.item.ItemStack;
+
+/**
+ * potions, in my mind atleast, are just water bottles with stuff in it.
```

Comment:
maybe only append `PotionFluidVolume` to normal water bottles for now until we do handling for potion nbts?

Reply:
I can just add fluid properties for the potion data